website/content/en/docs/faq.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "FAQs"
+linkTitle: "FAQs"
+weight: 30
+---
+
+## General
+### How does a Provisioner decide to manage a particular node?
+Each node will have a set of predetermined Karpenter labels. Provisioners will use `name` and `namespace` labels to distinguish between Provisioners. Furthermore, a Provisioner will only take action on a node based on the label that details what phase a node is in. e.g. a provisioner will only consider a node for termination if its phase label says "underutilized".
```

Comment:
Would you be able to change this to be the following?
> Provisioners will use a `name` label to distinguish between Provisioners

Reply:
ty